tests/integration/targets/yum_versionlock/tasks/main.yml

```
@@ -55,6 +67,8 @@
           assert:
             that:
               - "{{ lock_all_packages.changed }}"
+              - "{{ not lock_all_packages_again.changed }}"
```

```suggestion
              - lock_all_packages_again is not changed
```
Same for all other entries.